conf/modules.config

```diff
@@ -607,6 +607,9 @@ process{
         ext.prefix       = { meta.num_intervals <= 1 ? "${meta.id}" : "${meta.id}.${target_bed.simpleName}" }
         ext.args         = '--min-alternate-fraction 0.1 --min-mapping-quality 1'
         ext.when         = { params.tools && params.tools.split(',').contains('freebayes') }
+        publishDir       = [
```

Comment:
why is this happening though? shouldn't publishing be disabled by default?

Reply:
I agree but it was

Reply:
I believe you :D just wondering if something else is going on somewhere.

Reply:
I'm sure something else is going on. I'll check it more more in detail next week